src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetHolder.java

```diff
@@ -22,25 +22,22 @@
 import java.util.concurrent.Callable;
 
 public class FuzzTargetHolder {
-  public static final FuzzTarget AUTOFUZZ_FUZZ_TARGET;
-
-  static {
+  public static FuzzTarget autofuzzFuzzTarget(Callable<Object> newInstance) {
     try {
-      AUTOFUZZ_FUZZ_TARGET =
-          new FuzzTarget(com.code_intelligence.jazzer.autofuzz.FuzzTarget.class.getMethod(
-                             "fuzzerTestOneInput", FuzzedDataProvider.class),
-              ()
-                  -> {
-                com.code_intelligence.jazzer.autofuzz.FuzzTarget.fuzzerInitialize(
-                    Opt.targetArgs.toArray(new String[0]));
-                return null;
-              },
-              Optional.empty());
+      Method fuzzTestOneInput = com.code_intelligence.jazzer.autofuzz.FuzzTarget.class.getMethod(
+          "fuzzerTestOneInput", FuzzedDataProvider.class);
+      return new FuzzTargetHolder.FuzzTarget(fuzzTestOneInput, newInstance, Optional.empty());
```

Comment:
`fuzzerTestOneInput` instead of `fuzzTestOneInput` for consistency.

Reply:
Done.